src/app/app-routing.module.ts

```diff
@@ -14,7 +14,7 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 const routes: Routes = [
-    { path: '', redirectTo: '/channels', pathMatch: 'full' },
+    { path: '', redirectTo: '/campaigns', pathMatch: 'full' },
```

Comment:
This is unfortunately not correct. Default page should still be channels. David only wanted the Campaigns to be renamed to "Crowdfunding" in the header.